vidarr-prometheus/src/main/java/ca/on/oicr/gsi/vidarr/prometheus/AlertmanagerAutoInhibitConsumableResource.java

```diff
@@ -146,7 +146,7 @@ public ConsumableResourceResponse request(
       return ConsumableResourceResponse.error(
           String.format(
               "Workflow %s has been throttled by %s alert(s) for: %s",
-              alertName, workflowName, String.join(", ", isInhibited)));
+              workflowName, alertName, String.join(", ", isInhibited)));
```

Comment:
this took a bit to understand bc the name `isInhibited` implies a boolean, not a set, could we change this to something like `inhibitedWorkflows` while we're here?

Reply:
Yes, that makes sense! Since it will inhibit on `vidarr` as well, I'll use `matchedAlertValues`

Reply:
Since it inhibits on `vidarr` as well what do you think about `matchedAlertValues`?

Reply:
Sure!